airflow/providers/amazon/aws/sensors/batch.py

```diff
@@ -113,9 +114,10 @@ def execute_complete(self, context: Context, event: dict[str, Any]) -> None:
 
         Relies on trigger to throw an exception, otherwise it assumes execution was successful.
         """
-        if "status" in event and event["status"] == "failure":
```

Comment:
Are we no longer worried about an IndexError here if 'status' is not in event??

Reply:
no I don't think so. The event is something we control and that we send to ourselves. If it doesn't contain a "status" key, it means something bad happened, either the event is not formatted properly, or an event was routed to the wrong place, and we definitely want the error to blow up.